README.md

```diff
@@ -0,0 +1,19 @@
+# iExec-Solidity
+
+## Release new version on both Github and NPM
+
+### Steps :
+
+0. **Ask acces**: To be able to write on this repository
+
+1. **Prepare the Release**
+
+2. **Update the Version**: Modify the version number in the `package.json` file in line with the new release.
+
+3. **Commit the Version Change**
+
+4. **Tag the Release on GitHub**
+
+5. **Publish a new release on GitHub**
+
+6. **Publish the Release on NPM**: Before publishing, ensure you have access to the npm registry of iExec team. If you don't, request access from the admin. After that, you can use the `npm publish` command to release the new version on NPM.
```

Comment:
How do we authenticate to NPM registry?

Reply:
The npm publish command will display the link to authenticate